src/state/auth/reducer.js

```diff
@@ -97,15 +101,15 @@ export function authReducer(state = initialState, action) {
         ...payload,
       }
     case types.CLEAR_AUTH:
-      Authenticator.set({ ...initialAuthData, isPersisted: state.isPersisted })
+      Authenticator.clear()
```

Comment:
this seems to be a side effect and shouldn't be in reducer, we should move to saga later.